Packs/CheckPointNDR/Integrations/CheckPointNDR/CheckPointNDR.yml

```diff
@@ -0,0 +1,80 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: CheckPointNDR
+  version: -1
+configuration:
+- section: Connect
+  display: Infinity NDR API URL
+  name: url
+  defaultvalue: "https://portal.now.checkpoint.com"
+  type: 0
+  required: true
+- section: Connect
+  display: Client ID
+  displaypassword: Access Key
+  name: credentials
+  type: 9
+  required: true
+- section: Connect
+  display: NDR Domain name
+  name: domain
+  type: 0
+  required: true
+- section: Connect
+  advanced: true
+  display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- section: Connect
+  advanced: true
+  display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- section: Collect
+  display: First fetch time ("<number> <time unit>" e.g. "24 hours", "7 days")
+  name: first_fetch
+  defaultvalue: 7 days
+  type: 0
+  required: false
+- section: Collect
+  advanced: true
+  display: Fetch incidents
+  name: isFetch
+  defaultvalue: 'true'
+  type: 8
+  required: false
+- section: Collect
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  defaultvalue: "10"
+  type: 19
+  required: false
+- section: Collect
+  display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- section: Collect
+  display: Maximum number of incidents per fetch
+  name: max_fetch
```

Comment:
The API where the incidents are fetched from dont have an amount limit option (only time frame) so there is nowhere to use this parameter.
But I can't remove it from the YML because then  Demisto Validator gives an error as this is apparently a-must parameter here...

Reply:
Got it, but we still need to support it
And that is why it is not possible to remove it from the yml.
In cases where the api does not support the limit, you will have to handle it in our code, you will collect incidents until we reach the desired amount and then we will save the time or the id or any data that will help us in the next round to continue bringing where we left off, you just have to be very careful on the one hand not to miss And on the other hand, do not bring duplications
Let me know if you need any more help with this

Reply:
Ok